module/VuFind/src/VuFind/OAuth2/Repository/TokenRepositoryFactory.php

```diff
@@ -44,7 +44,11 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org Main Site
  */
+<<<<<<<< HEAD:module/VuFind/src/VuFind/OAuth2/Repository/TokenRepositoryFactory.php
```

Comment:
@ThoWagen, this is a very fresh commit, so I realize you may still be working on it, but in case you haven't already seen this, there appear to be conflict markers committed to the PR at this point in time. :-)

Reply:
Resolving the merge conflicts took some time so I forgot, to finish the merge request and just committed the changes. So the merge got messed up unfortunately. But it should be fixed now finally. 

Sorry for the hick up.

Reply:
No problem, it will all go away when we "squash and merge" at the end of the process. :-)